### DISCUSSION TITLE
RSRP at a specific receiver

### DISCUSSION THREAD

**Discussion by manoj-kumar-joshi** [NVlabs/sionna#208] [Q&A] [answered]
Although coverage map gives us an option select sample locations, I was wondering if I can get RSRP at a specific receiver from all the transmitters in the scene. My objective is to calculate interference and RSRQ. May be from paths?

**Comment by faycalaa** [comment, marked as answer]
Hi,

If the received signal power at a specific receiver position is needed, then computing the power from the paths would give a more accurate result.

A coverage map provides for each of its cell an estimate of the average power received over this cell, and is therefore only an approximation for any specific position.
